javascript/asynchronous/loops-and-intervals/setinterval-stopwatch.html

```diff
@@ -47,11 +47,13 @@
       // When the start button is pressed, start running displayCount() once per second using displayInterval()
       startBtn.addEventListener('click', () => {
         stopWatch = setInterval(displayCount, 1000);
+        startBtn.disabled =true;
       });
 
       // When the stop button is pressed, clear the interval to stop the count.
       stopBtn.addEventListener('click', () => {
         clearInterval(stopWatch);
+        startBtn.disabled=false;
```

Comment:
put space on both sides of the equials sign, i.e.

```
startBtn.disabled = false;
```

Reply:
startBtn.disabled = false;